### PR TITLE
[Fix Bugs] Update web_demo.py

### PR DESCRIPTION
**Pull request by crazysteeaam**
1. 实测 `from transformers import AutoModelForCausalLM, AutoTokenizer` 必须先引入 `AutoTokenizer` 再引入`AutoModelForCausalLM` 否则会报以下错

```bash
Traceback (most recent call last):
  File "/root/.conda/envs/InternLM/lib/python3.10/site-packages/streamlit/runtime/scriptrunner/script_runner.py", line 552, in _run_script
    exec(code, module.__dict__)
  File "/root/code/InternLM2/web_demo.py", line 10, in <module>
    from transformers import AutoModelForCausalLM, AutoTokenizer
ImportError: cannot import name 'AutoModelForCausalLM' from 'transformers' (/root/.local/lib/python3.10/site-packages/transformers/__init__.py)
load model begin.
```

2. 必须用 steamlit 命令，不能用 python 命令运行 web_demo.py 否则会报以下错

``` bash
2024-01-19 15:02:13.807 Session state does not function when running a script without `streamlit run`
Traceback (most recent call last):
  File "/root/.conda/envs/InternLM/lib/python3.10/site-packages/streamlit/runtime/state/session_state.py", line 378, in __getitem__
    return self._getitem(widget_id, key)
  File "/root/.conda/envs/InternLM/lib/python3.10/site-packages/streamlit/runtime/state/session_state.py", line 423, in _getitem
    raise KeyError
KeyError

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/root/.conda/envs/InternLM/lib/python3.10/site-packages/streamlit/runtime/state/session_state_proxy.py", line 119, in __getattr__
    return self[key]
  File "/root/.conda/envs/InternLM/lib/python3.10/site-packages/streamlit/runtime/state/session_state_proxy.py", line 90, in __getitem__
    return get_session_state()[key]
  File "/root/.conda/envs/InternLM/lib/python3.10/site-packages/streamlit/runtime/state/safe_session_state.py", line 113, in __getitem__
    return self._state[key]
  File "/root/.conda/envs/InternLM/lib/python3.10/site-packages/streamlit/runtime/state/session_state.py", line 380, in __getitem__
    raise KeyError(_missing_key_error_message(key))
KeyError: 'st.session_state has no key "messages". Did you forget to initialize it? More info: https://docs.streamlit.io/library/advanced-features/session-state#initialization'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "//root/code/InternLM2/web_demo.py", line 251, in <module>
    main()
  File "//root/code/InternLM2/web_demo.py", line 214, in main
    for message in st.session_state.messages:
  File "/root/.conda/envs/InternLM/lib/python3.10/site-packages/streamlit/runtime/state/session_state_proxy.py", line 121, in __getattr__
    raise AttributeError(_missing_attr_error_message(key))
AttributeError: st.session_state has no attribute "messages". Did you forget to initialize it? More info: https://docs.streamlit.io/library/advanced-features/session-state#initialization
```

Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

Please describe the motivation of this PR and the goal you want to achieve through this PR.

## Modification

Please briefly describe what modification is made in this PR.

## BC-breaking (Optional)

Does the modification introduce changes that break the backward compatibility of the downstream repositories?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here and update the documentation.

## Checklist

**Before PR**:

- [ ] Pre-commit or other linting tools are used to fix the potential lint issues.
- [ ] Bug fixes are fully covered by unit tests, the case that causes the bug should be added in the unit tests.
- [ ] The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
- [ ] The documentation has been modified accordingly, like docstring or example tutorials.

**After PR**:

- [ ] If the modification has potential influence on downstream or other related projects, this PR should be tested with those projects.
- [ ] CLA has been signed and all committers have signed the CLA in this PR.

### CODE REVIEW

**Review by ZwwWayne — APPROVED** [review]

**Comment by ZwwWayne** [comment]
Hi @crazysteeaam ,
Thank you for your kind PR. Could you fix the lint issue so that we can merge this PR?

**Comment by crazysteeaam** [comment]
> Hi @crazysteeaam , Thank you for your kind PR. Could you fix the lint issue so that we can merge this PR?

done

**Comment by hellock** [comment]
isort should be ignored and leave a comment there.

File: chat/web_demo.py [hunk 1]
```diff
@@ -5,8 +5,11 @@
     1. streamlit chat example: https://docs.streamlit.io/knowledge-base/tutorials/build-conversational-apps
     2. chatglm2: https://github.com/THUDM/ChatGLM2-6B
     3. transformers: https://github.com/huggingface/transformers
+Please run with the command `streamlit run path/to/web_demo.py --server.address=0.0.0.0 --server.port 7860`.
+Using `python path/to/web_demo.py` may cause unknown problems.
 """
 
+# isort: skip_file
```
Comment (hellock) [inline chat/web_demo.py:12]: Do not skip the whole file, just the conflicted line.

**Comment by hellock** [comment]
The linting still fails.

**Pull request merged**